manage.py

```diff
@@ -34,17 +34,27 @@ def mail_certificate(id, email):
                 recipients=[email]
                 )
         msg.body = render_template('mail.txt')
-        certificate_path = "{}/freifunk_{}.tgz".format(
-                app.config['DIRECTORY_CLIENTS'],
-                id
-                )
-        with app.open_resource(certificate_path) as fp:
-            msg.attach(
-                    "freifunk_{}.tgz".format(id),
-                    "application/gzip",
-                    fp.read()
+        print("Looking for archive to attach...")
+        try:
+            certificate_path = "{}/freifunk_{}.tgz".format(
+                    app.config['DIRECTORY_CLIENTS'],
+                    id
                     )
-        mail.send(msg)
+            with app.open_resource(certificate_path) as fp:
+                msg.attach(
+                        "freifunk_{}.tgz".format(id),
+                        "application/gzip",
+                        fp.read()
+                        )
+        except:
+            print("Sorry, that didn't work.")
```

Comment:
We should inform the user what went wrong. Otherwise we could also just print the stacktrace and confuse the user even more.